examples/prometheus/prometheus.yaml

```diff
@@ -291,3 +301,196 @@ objects:
         - source_labels: [__meta_kubernetes_service_name]
           action: replace
           target_label: kubernetes_name
+
+      alerting:
+        alertmanagers:
+        - scheme: http
+          static_configs:
+          - targets:
+            - "alertmanager:9093"
+
+# Create a fully end-to-end TLS connection to the proxy
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: alerts
+    namespace: "${NAMESPACE}"
+  spec:
+    to:
+      name: alerts
+    tls:
+      termination: Reencrypt
+
+# Temporary route to alertmanager
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: alertmanager
+    namespace: "${NAMESPACE}"
+  spec:
+    port:
+      targetPort: alertmanager
+    to:
+      kind: Service
+      name: alertmanager
+      weight: 100
+    wildcardPolicy: None
+
+# Create a service to access alerts vi auth-proxy
+- apiVersion: v1
+  kind: Service
+  metadata:
+    annotations:
+      service.alpha.openshift.io/serving-cert-secret-name: alerts-tls
+    labels:
+      name: alerts
+    name: alerts
+    namespace: "${NAMESPACE}"
+  spec:
+    ports:
+    - name: alerts
+      port: 443
+      protocol: TCP
+      targetPort: 8443
+    selector:
+      app: alerts
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: alerts-proxy
+    namespace: "${NAMESPACE}"
+  stringData:
+    session_secret: "${SESSION_SECRET}="
+
+# Create an internal direct service to alertmanager app
+- apiVersion: v1
```

Comment:
The main reason for separation to 2 pods is that we might want to scale the prometheus pods.
The question is what is the best way to secure the alertmanager service?